Active_Record/app/models/sup.rb

```diff
@@ -0,0 +1,10 @@
+class Sup < ActiveRecord::Base
+  has_many :customers, through: :bookings
+  has_many :bookings
```

Comment:
`has_many through` should be AFTER `has_many`, which declares   joiining relation

Reply:
done